components/Repo/Phases.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import { css, merge } from 'glamor'
+import { Link } from '../../lib/routes'
+import { withRouter } from 'next/router'
+import { fontStyles } from '@project-r/styleguide'
+
+const styles = {
+  phase: css({
+    color: '#fff',
+    padding: '5px 10px',
```

Comment:
I actually prefer the padding of the current Phase component: '3px 6px' a bit more, makes it a bit more condensed.